ui/desktop/app/templates/scopes/scope/projects/targets/index.hbs

```diff
@@ -13,189 +13,149 @@
-                            @selectedItems={{filter.selectedValue}}
-                            @onChange={{route-action 'filterBy' filter.name}}
-                            as |group|
-                          >
-                            <li>
-                              <dropdown.item>
-                                <group.checkbox
-                                  @label={{group.item.displayName}}
-                                  value={{group.item}}
-                                />
-                              </dropdown.item>
-                            </li>
-                          </form.checkboxGroup>
-                        </ul>
-                      </li>
-                    {{/each}}
-                  {{/let}}
-                </ul>
-              </Rose::Dropdown>
-            {{/let}}
-
-            {{#if
-              (has-resource-filter-selections
-                'scopes.scope.projects.targets' 'project'
-              )
-            }}
-              <Rose::Button
-                @style='inline-link-action'
-                @iconLeft='flight-icons/svg/x-16'
-                {{on 'click' (route-action 'clearAllFilters')}}
-              >{{t 'actions.clear-all-filters'}}
-              </Rose::Button>
-            {{/if}}
-          </Rose::Form>
-        </toolbar.primary>
-
-        <toolbar.secondary>
-          <ToolbarRefresher @onClick={{route-action 'refreshTargets'}} />
-        </toolbar.secondary>
-
-      </Rose::Toolbar>
-    {{/if}}
-
+  <page.body class='targets'>
     {{#if @model}}
-
-      <Hds::Table
-        @model={{@model}}
-        @columns={{array
-          (hash label=(t 'form.name.label'))
-          (hash label=(t 'form.id.label'))
-          (hash label=(t 'form.type.label'))
-          (hash label=(t 'resources.project.title'))
-          (hash label='')
-        }}
-        @valign='middle'
-      >
-        <:body as |B|>
-          <B.Tr>
-            <B.Td>
-              <span class='hds-font-weight-semibold'>
-                {{#if (can 'read model' B.data)}}
-                  <LinkTo
-                    data-test-visit-target={{B.data.id}}
-                    @route='scopes.scope.projects.targets.target'
-                    @model={{B.data.id}}
-                    @query={{hash isConnecting=false}}
-                  >
+      <div class='target-search-filtering'>
+        <Hds::SegmentedGroup as |S|>
+          <S.TextInput
+            @type='search'
+            placeholder={{t 'actions.search'}}
+            aria-label={{t 'actions.search'}}
+            {{on 'input' this.handleInput}}
+          />
+          <S.Dropdown as |DD|>
+            <DD.ToggleButton
+              @color='secondary'
+              @text={{t 'resources.scope.title'}}
+            />
+            <DD.Header @hasDivider={{true}}>
+              <Hds::Form::TextInput::Base
+                @type='search'
+                placeholder={{t 'actions.narrow-results'}}
```

Comment:
? for learning: why doesn't this input need an `aria-label` but the input in line 20 does?

Reply:
Answer for learning: It actually does need `aria-label`. I just missed it here. Looking [here](https://helios.hashicorp.design/components/form/text-input?tab=code#formtextinputbase) at the HDS docs for base text input, we are responsible for making sure it is added. Thanks for catching. Fixed now.